services/blockchain-indexer/shared/dataService/business/transactionsEstimateFees.js

```diff
@@ -193,26 +203,53 @@ const estimateTransactionFees = async params => {
 
 	const trxWithMockProps = await mockTransaction(params.transaction, authAccountInfo);
 	const formattedTransaction = await requestConnector('formatTransaction', { transaction: trxWithMockProps });
+	const feeEstimatePerByte = await requestFeeEstimator('estimates');
```

Comment:
Can we please cache the estimation values and update it based on update (moleculer) events from the fees microservice? Lets connect offline to discuss the specifics.
```suggestion
	const feeEstimatePerByte = await requestFees('estimates');
```